### PR TITLE
Update documentation for PortableSharedCache on Z and Power

### PR DESCRIPTION
**Pull request by harryyu1994**
Issue: https://github.com/eclipse/openj9-docs/issues/651

Signed-off-by: Harry Yu <harryyu1994@gmail.com>

### CODE REVIEW

**Comment by harryyu1994** [comment]
@mpirvu FYI

**Comment by harryyu1994** [comment]
@pmhayward Ready for review

**Comment by harryyu1994** [comment]
Actually putting this on hold again as the mentioned updates did not make into 0.23 release.

**Review by mpirvu — APPROVED** [review]
LGTM

**Comment by mpirvu** [comment]
oh, just read the update. So the changes for Z were not included before the code split. In that case, yes we should not mention about them

**Review by mpirvu — COMMENTED** [review]
Indeed the enablement for Z happened 5 days ago while the split happened 7 days ago, so these changes should not be included in the 0.23 release

**Comment by harryyu1994** [comment]
@pmhayward @mpirvu The changes to enable Portable Shared Cache on Z and Power have now been merged to OpenJ9. This one is ready for review.

File: docs/version0.24.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# What's new in version 0.24.0
+
+## Features and changes
+
+### Binaries and supported environments
+
+OpenJ9 release 0.23.0 supports OpenJDK 8, 11, and 15. Binaries are available from the AdoptOpenJDK community at the following links:
```
Comment (mpirvu) [inline docs/version0.24.md:31]: Typo, the release should be 0.24.0

**Review by mpirvu — APPROVED** [review]
LGTM

**Comment by pmhayward** [comment]
Jenkins doc stage

**Comment by pmhayward** [comment]
Jenkins doc stage

**Pull request merged**